### PR TITLE
prebuild CSS/JS before xgo release binaries

### PR DESCRIPTION
**Pull request by techknowlogick**
Resolve failed build: https://drone.gitea.io/go-gitea/gitea/17124/3/3

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -388,6 +388,19 @@ steps:
...
       - git fetch --tags --force
 
+  - name: build
+    pull: always
+    image: golang:1.13
+    commands:
+      - curl -sL https://deb.nodesource.com/setup_12.x | bash - && apt -y install nodejs
+      - make css
+      - make js
+      - make generate
```
Comment (lafriks) [inline .drone.yml]: Didn't we get rid of make generate?

Comment (techknowlogick) [reply .drone.yml]: `make build` runs make generate before building, but in the case of xgo it was included in the `release` step I just moved it back out again.

Thinking about this, I could probably put it back in release, and change the image that css/js use to speed up build.

Comment (techknowlogick) [reply .drone.yml]: Please see update commits for changes.

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
Do this need a backport to 1.10 ?

**Comment by techknowlogick** [comment]
@6543 no, this is due to refactoring done on 1.11-dev only.

**Comment by 6543** [comment]
O right the remove of js and css files was on master only currently ...

**Review by axifive — APPROVED** [review]

**Pull request merged**